src/fish_ai/config.py

```diff
@@ -2,10 +2,12 @@
 from os import path
 import sys
 from configparser import ConfigParser
+import fish_ai.cred as cred
 
 config = ConfigParser()
 config.read(path.expanduser('~/.config/fish-ai.ini'))
-
+use_keyring = config.getboolean(section='fish-ai', option='use_keyring',
```

Comment:
I would prefer to not have a separate option for this, and just attempt to load the credential from the keyring if the API key is omitted from the configuration.